### ISSUE TITLE
Landing page: Add correct heading level in footer

### ISSUE THREAD

**Issue by jennur** [inveniosoftware/invenio-app-rdm#1271]
The headers in the footer columns are the wrong heading level (h4).

The headings should be h2, but keep the same size.

<img width="541" alt="Screenshot 2022-02-16 at 10 48 33" src="https://user-images.githubusercontent.com/21052053/154239160-0c5f9147-f350-4d94-8a37-4f0f835b5bad.png">
